File: AMS_电子对账/code/server/source/com.yss.uco.elecreco/src/main/java/com/yss/uco/elecreco/er/erzcfzb/pojo/ErZcfzb.java
package com.yss.uco.elecreco.er.erzcfzb.pojo;

import com.yss.framework.api.common.co.BasePojo;

public class ErZcfzb extends BasePojo {

	/**
	 * 
	 */
	private static final long serialVersionUID = 1L;
	
	/**
	 * 报文序号
	 */
	private String c_SN = "";

	/**
	 * 资金代码
	 */
	private String c_ASS_CODE = "";

	/**
	 * 文件类型
	 */
	private String c_FILE_TYPE ="";

	/**
	 * 报表类型
	 */
	private String c_RPT_TYPE ="";

		/**
	 * 开始日期
	 */
	private String d_START_DATE = "";

	/**
	 * 结束日期
	 */
	private String d_END_DATE = "";

	/**
	 * 公司代码
	 */
	private String c_DEPT_CODE = "";
	
	/**
	 * 证书代码
	 */
	private String c_CERT_ID = "";

	/**
	 * 指标代码
	 */
	private String c_INDEX_CODE = "";
	
	/**
	 * 指标名称
	 */
	private String c_INDEX_NAME = "";
	
	/**
	 * 期初值
	 */
	private String n_BEGIN_VALUE = "";
	
	/**
	 * 期末值
	 */
	private String n_END_VALUE = "";

	public String getC_SN() {
		return c_SN;
	}

	public void setC_SN(String c_SN) {
		this.c_SN = c_SN;
	}

	public String getC_ASS_CODE() {
		return c_ASS_CODE;
	}

	public void setC_ASS_CODE(String c_ASS_CODE) {
		this.c_ASS_CODE = c_ASS_CODE;
	}

	public String getC_FILE_TYPE() {
		return c_FILE_TYPE;
	}

	public void setC_FILE_TYPE(String c_FILE_TYPE) {
		this.c_FILE_TYPE = c_FILE_TYPE;
	}

	public String getC_RPT_TYPE() {
		return c_RPT_TYPE;
	}

	public void setC_RPT_TYPE(String c_RPT_TYPE) {
		this.c_RPT_TYPE = c_RPT_TYPE;
	}

	public String getD_START_DATE() {
		return d_START_DATE;
	}

	public void setD_START_DATE(String d_START_DATE) {
		this.d_START_DATE = d_START_DATE;
	}

	public String getD_END_DATE() {
		return d_END_DATE;
	}

	public void setD_END_DATE(String d_END_DATE) {
		this.d_END_DATE = d_END_DATE;
	}

	public String getC_DEPT_CODE() {
		return c_DEPT_CODE;
	}

	public void setC_DEPT_CODE(String c_DEPT_CODE) {
		this.c_DEPT_CODE = c_DEPT_CODE;
	}

	public String getC_CERT_ID() {
		return c_CERT_ID;
	}

	public void setC_CERT_ID(String c_CERT_ID) {
		this.c_CERT_ID = c_CERT_ID;
	}

	public String getC_INDEX_CODE() {
		return c_INDEX_CODE;
	}

	public void setC_INDEX_CODE(String c_INDEX_CODE) {
		this.c_INDEX_CODE = c_INDEX_CODE;
	}

	public String getN_BEGIN_VALUE() {
		return n_BEGIN_VALUE;
	}

	public void setN_BEGIN_VALUE(String n_BEGIN_VALUE) {
		this.n_BEGIN_VALUE = n_BEGIN_VALUE;
	}

	public String getN_END_VALUE() {
		return n_END_VALUE;
	}

	public void setN_END_VALUE(String n_END_VALUE) {
		this.n_END_VALUE = n_END_VALUE;
	}

	public String getC_INDEX_NAME() {
		return c_INDEX_NAME;
	}

	public void setC_INDEX_NAME(String c_INDEX_NAME) {
		this.c_INDEX_NAME = c_INDEX_NAME;
	}

}

File: AMS_电子对账/code/source/YssElecReco/Pojo/Er/ErZcfzb.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Drawing;
using System.Linq;
using System.Reflection;
using System.Windows.Forms;

using Newtonsoft.Json;
using FAST.Common.Service.Pojo.Base;

namespace YssElecReco.pojo.Er
{
    /// <summary>
    /// 资产负债表
    /// </summary>
    public class ErZcfzb : BasePojo
    {
        /// <summary>
        /// 报文序号
        /// </summary>
        private string c_SN = "";

        /// <summary>
        /// 资产代码
        /// </summary>
        private string c_ASS_CODE = "";

        /// <summary>
        /// 文件类型
        /// </summary>
        private string c_FILE_TYPE = "";

        /// <summary>
        /// 报表类型
        /// </summary>
        private string c_RPT_TYPE = "";

        /// <summary>
        /// 开始日期
        /// </summary>
        private string d_START_DATE = "";

        /// <summary>
        /// 结束日期
        /// </summary>
        private string d_END_DATE = "";

        /// <summary>
        /// 公司代码
        /// </summary>
        private string c_DEPT_CODE = "";

        /// <summary>
        /// 证书代码
        /// </summary>
        private string c_CERT_ID = "";

        /// <summary>
        /// 指标代码
        /// </summary>
        private string c_INDEX_CODE = "";

        /// <summary>
        /// 指标名称
        /// </summary>
        private string c_INDEX_NAME = "";

        /// <summary>
        /// 期初值
        /// </summary>
        private decimal n_BEGIN_VALUE = 0;

        /// <summary>
        /// 期末值
        /// </summary>
        private decimal n_END_VALUE = 0;

        /// <summary>
        /// 报文序号
        /// </summary>
        [JsonProperty(PropertyName = "c_SN")]
        public string C_SN
        {
            get { return c_SN; }
            set { c_SN = value; }
        }

        /// <summary>
        /// 属性: 资产代码 
        /// </summary>
        [JsonProperty(PropertyName = "c_ASS_CODE")]
        public string C_ASS_CODE
        {
            set { c_ASS_CODE = value; }
            get { return c_ASS_CODE; }
        }

        /// <summary>
        /// 文件类型
        /// </summary>
        [JsonProperty(PropertyName = "c_FILE_TYPE")]
        public string C_FILE_TYPE
        {
            set { c_FILE_TYPE = value; }
            get { return c_FILE_TYPE; }
        }

        /// <summary>
        /// 报表类型
        /// </summary>
        [JsonProperty(PropertyName = "c_RPT_TYPE")]
        public string C_RPT_TYPE
        {
            set { c_RPT_TYPE = value; }
            get { return c_RPT_TYPE; }
        }

        /// <summary>
        /// 开始日期
        /// </summary>
        [JsonProperty(PropertyName = "d_START_DATE")]
        public string D_START_DATE
        {
            get { return d_START_DATE; }
            set { d_START_DATE = value; }
        }

        /// <summary>
        /// 结束日期
        /// </summary>
        [JsonProperty(PropertyName = "d_END_DATE")]
        public string D_END_DATE
        {
            get { return d_END_DATE; }
            set { d_END_DATE = value; }
        }

        /// <summary>
        ///  公司代码
        /// </summary>
        [JsonProperty(PropertyName = "c_DEPT_CODE")]
        public string C_DEPT_CODE
        {
            get { return c_DEPT_CODE; }
            set { c_DEPT_CODE = value; }
        }

        /// <summary>
        /// 证书代码
        /// </summary>
        [JsonProperty(PropertyName = "c_CERT_ID")]
        public string C_CERT_ID
        {
            get { return c_CERT_ID; }
            set { c_CERT_ID = value; }
        }

        /// <summary>
        /// 指标代码
        /// </summary>
        [JsonProperty(PropertyName = "c_INDEX_CODE")]
        public string C_INDEX_CODE
        {
            get { return c_INDEX_CODE; }
            set { c_INDEX_CODE = value; }
        }

        /// <summary>
        /// 指标名称
        /// </summary>
        [JsonProperty(PropertyName = "c_INDEX_NAME")]
        public string C_INDEX_NAME
        {
            get { return c_INDEX_NAME; }
            set { c_INDEX_NAME = value; }
        }

        /// <summary>
        /// 期初值
        /// </summary>
        [JsonProperty(PropertyName = "n_BEGIN_VALUE")]
        public decimal N_BEGIN_VALUE
        {
            get { return n_BEGIN_VALUE; }
            set { n_BEGIN_VALUE = value; }
        }

        /// <summary>
        /// 期末值
        /// </summary>
        [JsonProperty(PropertyName = "n_END_VALUE")]
        public decimal N_END_VALUE
        {
            get { return n_END_VALUE; }
            set { n_END_VALUE = value; }
        }
    }
}
